iteration_test.go

```diff
@@ -284,6 +285,7 @@ func assertIterationLinking(t *testing.T, target *app.Iteration) {
 	assert.NotNil(t, target.Relationships)
 	assert.NotNil(t, target.Relationships.Space)
 	assert.NotNil(t, target.Relationships.Space.Links.Self)
+	assert.True(t, strings.Contains(*target.Relationships.Space.Links.Self, "/api/spaces/"))
```

Comment:
Please make all `assert` calls before a `require` so we don't access `nil` memory. Otherwise this looks good.